modules/standard/VisualDebug.chpl

```diff
@@ -31,6 +31,10 @@ module VisualDebug
 
   use String;
 
+  // VDebugOn -- Should we generate data
+  config param DefaultVDebugOn = true;
+  config var VDebugOn = DefaultVDebugOn;
```

Comment:
Yeah, I can appreciate the name match.  My thinking is that you don't see startVdebug() etc. in a source file without a 'use VisualDebug' being there already.  But the config param might be mentioned on the compile command line, where there isn't any other context for its meaning.  So there, making it more self-explanatory is good.